service.go

```diff
@@ -20,25 +20,26 @@ const (
 	WatchTimeOut = 90 * time.Second
 )
 
-// Config represents the configuration used to create a service.
-type Config struct {
+// ServiceConfig represents the configuration used to create a certificate TPR
+// service.
+type ServiceConfig struct {
```

Comment:
Here happened some renaming which makes the naming more correct based on our current standards.